docs/concepts/07_dip/03_consumer.md

```diff
@@ -0,0 +1,88 @@
+---
+id: consumer-pallet
+title: Consumer pallet
+---
+
+This pallet is a component of the DIP protocol.
+It enables entities with an identity on a connected Substrate-based chain (provider) to use those identities on the chain this pallet is deployed (consumers) without requiring those entities to set up a new identity locally.
+
+A consumer chain is _connected_ to a provider if there is a way for the consumer chain to verify state proofs about parts of the state of the provider chain.
+
+A cross-chain transaction with DIP assumes the entity submitting the transaction has already generated a cross-chain identity commitment on the provider chain, by interacting with the DIP provider pallet on the provider chain.
+With a generated identity commitment, a cross-chain transaction flow for a generic entity `A` works as follows:
+
+1. `A` generates a state proof proving the state of the identity commitment on the provider chain.
+2. `A` generates any additional information required for the consumer runtime to successfully verify an identity proof.
+3. `A`, using their account `AccC` on the consumer chain, calls the `dispatch_as` extrinsic by providing its identifier on the provider chain, the generated proof, and the `Call` to be dispatched on the consumer chain.
```

Comment:
What is the meaning of `AccC`? Should there be a `C` indicator to denote the consumer chain?